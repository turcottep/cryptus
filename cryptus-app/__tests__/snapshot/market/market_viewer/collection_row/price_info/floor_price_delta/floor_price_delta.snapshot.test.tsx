import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import FloorPriceDelta from "../../../../../../../components/market/market_viewer/collection_row/price_info/floor_price_delta/floor_price_delta";

describe("<FloorPriceDelta />", () => {
  it("should display properly", async () => {
    const test_price = 7.79;
    const test_delta = 0.01;
    const test_curr = "eth";

    const tree = renderer
      .create(
        <FloorPriceDelta
          currency={test_curr}
          initial_price={0}
          current_price={0}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
