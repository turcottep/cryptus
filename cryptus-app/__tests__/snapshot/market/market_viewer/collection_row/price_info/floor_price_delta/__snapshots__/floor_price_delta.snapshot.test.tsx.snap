// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FloorPriceDelta /> should display properly 1`] = `
<div>
  0.0Îž (NaN%)
</div>
`;
